test/puppeteer_utils.js

```diff
@@ -99,291 +98,8 @@ describe('Apify.utils.puppeteer', () => {
-
-                expect(enqueued).to.have.lengthOf(3);
-
-                expect(enqueued[0].url).to.be.eql('https://example.com/a/b/first');
-                expect(enqueued[0].method).to.be.eql('POST');
-                expect(enqueued[0].userData).to.be.eql({ hello: 'world' });
-
-                expect(enqueued[1].url).to.be.eql('https://example.com/a/b/third');
-                expect(enqueued[1].method).to.be.eql('POST');
-                expect(enqueued[1].userData).to.be.eql({ hello: 'world' });
-
-                expect(enqueued[2].url).to.be.eql('http://cool.com/');
-                expect(enqueued[2].method).to.be.eql('GET');
-                expect(enqueued[2].userData).to.be.eql({ hello: 'world', foo: 'bar' });
-            } finally {
-                log.warning = originalLogWarning;
-            }
-
-            expect(logOutput.length).to.be.eql(1);
-            expect(logOutput[0]).to.include('options object');
-        });
-        it('throws with sparse pseudoUrls[]', async () => {
-            const enqueued = [];
-            const requestQueue = new RequestQueue('xxx');
-            requestQueue.addRequest = async (request) => {
-                enqueued.push(request);
-            };
-            const pseudoUrls = [
-                new Apify.PseudoUrl('https://example.com/[(\\w|-|/)*]', { method: 'POST' }),
-                null,
-                new Apify.PseudoUrl('[http|https]://cool.com/', { userData: { foo: 'bar' } }),
-            ];
-
-            try {
-                await Apify.utils.puppeteer.enqueueLinks({ page, selector: '.click', requestQueue, pseudoUrls });
-                throw new Error('Wrong error.');
-            } catch (err) {
-                expect(err.message).to.include('pseudoUrls[1]');
-                expect(enqueued).to.have.lengthOf(0);
-            }
-        });
-
-        it('DEPRECATED: enqueueRequestsFromClickableElements()', async () => {
-            const enqueuedUrls = [];
-            const queue = new RequestQueue('xxx');
-            queue.addRequest = (request) => {
-                expect(request.method).to.be.eql('POST');
-                enqueuedUrls.push(request.url);
-
-                return Promise.resolve();
-            };
-            const purls = [
-                new Apify.PseudoUrl('https://example.com/[(\\w|-|/)*]'),
-                new Apify.PseudoUrl('[http|https]://cool.com/'),
-            ];
-
-            await Apify.utils.puppeteer.enqueueRequestsFromClickableElements(page, '.click', purls, queue, { method: 'POST' });
-
-            expect(enqueuedUrls).to.be.eql([
-                'https://example.com/a/b/first',
-                'https://example.com/a/b/third',
-                'http://cool.com/',
-            ]);
-        });
+    it('enqueueLinks() exists in this namespace', async () => {
+        expect(Apify.utils.puppeteer.enqueueLinks).to.be.a('function');
```

Comment:
Maybe add a comment here that it used to be there in the legacy version and some people use it there, in half a year we'll not remember. Even better would be to print a single deprecation warning if people use this variant.